File: Java/Car.java
package Java;

public class Car {
  Integer id;
  String license;
  Driver driver;
  private Integer passenger;

  public Car( String license, Driver driver ){
    this.license = license;
    this.driver = driver;
  }
  public Integer getPassenger() {
    return passenger;
  }
  public void setPassenger(Integer passenger) {
    this.passenger = passenger;
  }

  void printDataCar(){
    System.out.println("License: " + this.license + " Name Driver: " + this.driver.name + " Passenger: " + passenger);
  }
}

File: PHP/Car.php
<?php
require_once('Account.php');
class Car{
  public $id;
  public $license;
  public $driver;
  public $passenger;

  public function __construct( string $license, Account $driver ){
    $this->license = $license;
    $this->driver = $driver;
  }
  public function printDataCar()
  {
    echo "License: $this->license Conductor name: {$this->driver->name}";
  }
}